### ISSUE TITLE
Rule Change: no-invalid-properties incorrectly reports for valid CSS with var() in functions

### ISSUE THREAD

**Issue by thecalamiity** [eslint/css#223]
### What rule do you want to change?

no-invalid-properties

### What change do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new default behavior

### Example code

```css
:root {
  --width: 10px;
}

div {
  width: calc(100% - var(--width))
}
```

### What does the rule currently do for this code?

The rule incorrectly reports: `Unknown property 'width' found`. This is a regression in the main branch, as the same code works correctly in v0.10.0.

### What will the rule do after it's changed?

The rule should recognize that `width` is a valid CSS property and that `calc(100% - var(--width))` is a valid value for the width property. It should not report any errors for this code.

### Participation

- [x] I am willing to submit a pull request to implement this change.

**Comment by nzakas** [comment]
Sounds good to me. :+1:
